public/views/answers.html

```diff
@@ -27,25 +27,24 @@
 
 <div id="info-container" ng-show="game.state === 'awaiting players'">
   <div id="inner-info">
-    <div id="lobby-how-to-play">How To Play</div>
-    <ol id="oh-el">
-      <li>Each player begins with, and will always have, 10 white answer cards.</li>
-      <li>For each round, one player is randomly chosen as the Card Czar.</li>
-      <li>Everyone else answers the black question card by clicking on the answer card they want to use.</li>
-      <li>The Card Czar then picks a favorite answer, and whoever played that answer wins the round.</li>
-      <li>*Want to adorn your avatar with a glorious crown? Donate to charity after the game!</li>
-    </ol>
+    <div id="lobby-how-to-play" class="col-md-offset-3" style="color: #ffffff">
+      <h4>HANG ON</h4>
+      <br/>
+      <h6>We are finding you people to play with :)</h6>
```

Comment:
Check this out: http://fontawesome.io/icon/smile-o/
For the smile
<i class="fa fa-smile-o" aria-hidden="true"></i>